app/src/main/kotlin/it/vfsfitvnm/vimusic/ui/components/themed/MediaItemMenu.kt

```diff
@@ -147,16 +147,6 @@ fun NonQueuedMediaItemMenu(
     BaseMediaItemMenu(
         mediaItem = mediaItem,
         onDismiss = onDismiss ?: menuState::hide,
-        onStartRadio = {
-            binder?.stopRadio()
-            binder?.player?.forcePlay(mediaItem)
-            binder?.setupRadio(
```

Comment:
if we don't use radio anymore, maybe we can remove the whole radio functionality?
Like `binder?.setupRadio` and `binder?.stopRadio()` and everything what is related to radio.

Reply:
I guess, these methods used in another sections. For example, in playlists folder these methods used to play all songs. I only deleted buttons in menu item and artist page

Reply:
got it!